src/components/Button/src/Button.vue

```diff
@@ -283,10 +294,32 @@ export default {
 			assert.error(['squared', 'rounded', 'pill'].includes(shapeValue), `${shapeValue} resolved from ${shapeValueOrPointer} is not a valid shape value and cannot be used in the shape prop of Button`);
 			return shapeValue;
 		},
+		resolvedFullWidth() {
```

Comment:
It's not clear to me why full width or align needs to be global default themeable properties. We don't have a current need for this and I can't imagine a future scenario where this configuration would be necessary.

Reply:
I don't feel strongly either way about `fullWidth` and `align` being themeable or not, but this was originally requested by @laurenhu so maybe she can speak to it

Reply:
I think it makes documentation for theming confusing if themes can access some props that control the UI for a given component but not others. Practically speaking, it means that developers using Maker components will need to be aware of applying some props while being able to ignore others (`fullWidth` on `ActionBarButton` being an example)

Reply:
There's several components where the defaults are sufficient and any props to change the behavior of something is the exception to the rule meant for limited use cases. Theming is meant to be a global configuration, props on components are local configurations. There's certainly overlap in terms of global and local options, but they do not need to be 1-to-1.  As a consumer no one should have to go to a global configuration on a theme in order to customize the behavior of a components local behavior in a view.  If all buttons have the same shape that's a global configuration that can be set once. If one button is customized to have a different shape that's an exception and set locally. If every prop on every component is a global configuration then it's more confusing where you're supposed to configure things. What's themeable globally is the responsibility of a smaller subset of engineers. Everyone else should be agnostic to it and define their rules as if the theme didn't exist with local overrides as exceptions.

Reply:
> As a consumer no one should have to go to a global configuration on a theme in order to customize the behavior of a components local behavior in a view.

This is still true even if we make `fullWidth` and `align` themeable props.

> If all buttons have the same shape that's a global configuration that can be set once. If one button is customized to have a different shape that's an exception and set locally.

Sounds very simple, intuitive, and straight-forward to me.

> If every prop on every component is a global configuration then it's more confusing where you're supposed to configure things. 

Yet you just explained how simple, intuitive, and straight-forward it was. How is it confusing?

> What's themeable globally is the responsibility of a smaller subset of engineers. 

This is still true even if we make `fullWidth` and `align` themeable props.

> Everyone else should be agnostic to it and define their rules as if the theme didn't exist with local overrides as exceptions.

This is still true even if we make `fullWidth` and `align` themeable props.

> [Our goal] should be to reduce the number of local props are set where in the global config works for the majority of use cases for the entire application.

This is still true even if we make `fullWidth` and `align` themeable props.